examples/applications.ron

```diff
@@ -0,0 +1,16 @@
+Config(
+  // Limit amount of entries shown by the applications plugin (default: 5)
+  max_entries: 5,
+  // Whether to evaluate desktop actions as well as desktop applications (default: false)
+  desktop_actions: false,
+  // Whether to use a specific terminal or just the first terminal available (default: None)
+  terminal: None,
+  // Whether or not to put more often used applications higher in the search rankings (default: true)
+  use_usage_statistics: true,
+  // How much score to add for every usage of an application (default: 50)
+  // Each matching letter is 25 points
+  usage_score_multiplier: 50,
+  // Maximum amount of usages to count (default: 10)
+  // This is to limit the added score, so often used apps don't get too big of a boost
+  max_counted_usages: 10,
```

Comment:
The benefit is just over the applications that haven't been used at all.
Technically, it would be enough to be able to mark an application as favorite, to give it a static boost over other apps.
Since I did not want to add interactivity to the plugin (because it's massively more complicated and would entail editing the base application), I chose to use a simple algorithm that would count up to 10 usages to give an app the full bonus score.

This way, you can accidentally launch an app a few times, without it impacting the score too much.
There are probably more advanced ways to do this, but I only spent an evening on this and it's good enough for my current purpose.

In another comment, you mentioned logarithms. I like the idea of capping the upper limit that way, but there would still have to be an upper limit, as to not overwhelm the actual search matches from the fuzzymatcher.